### PR TITLE
Use openshift-ansible DNS role

### PR DESCRIPTION
**Pull request by jprovaznik**
Switch to openshift-ansible for setting up DNS.

### CODE REVIEW

**Comment by markllama** [comment]
Would it make sense to change the `skip_dns` flag to something like `local_dns` and invert the value?

It would seem that a switch to tell the template what you want it to do rather than what you want it not to do would be clearer.

Just for my information: are there other DNS options than `skip_dns: [true|false]`? If `true`, are there other DNS options?

**Comment by jprovaznik** [comment]
I think both skip_dns and local_dns are quite equal regarding the clarity. TBH I'm not sure if I would know (as a user) what to imagine under "local_dns" parameter.

There are not other dns params except dns_nameservers. It might be that in future we will add support for creating designate records explicitly in temlates instead of relying on designate-sink. So in that far future we can replace true|false switch with multiple values selection.

File: infra.yaml [hunk 1]
```diff
@@ -267,6 +262,9 @@ parameters:
     description: List of repository URLs which will be installed on each node.
     default: ''
 
+  dns_forwarders:
+    type: comma_delimited_list
+    description: address of a dns nameserver reachable in your environment
```
Comment (lebauce) [inline infra.yaml]: nit: address of dns nameservers reachable in your environment

File: fragments/infra-boot.sh [hunk 2]
```diff
@@ -163,12 +141,6 @@ function verify_os_collect_config_is_installed() {
         notify_failure "os-collect-config service is not installed or enabled"
 }
 
-function install_enable_start_dnsmasq() {
```
Comment (lebauce) [inline fragments/infra-boot.sh:166]: The line 

```
use_local_dns && install_enable_start_dnsmasq
```

should also be removed

**Comment by lebauce** [comment]
The file `fragments/dnsmasq.conf` can probably be removed too

**Comment by jprovaznik** [comment]
fixed

**Pull request merged**